Cargo.toml

```diff
@@ -50,6 +50,7 @@ once_cell = { version = "1.4.0", features = ["parking_lot"] }
 base64 = "0.12.1"
 strum = { version = "0.18.0", features = ["derive"] }
 lol_html = "0.2"
+font-awesome-as-a-crate = "0.1.0"
```

Comment:
I'd prefer if we didn't take this approach and vendored as normal, it's needless complexity to maintain as well as being fragile in the first place

The more I think about it the less I like it, I'd really prefer going back to in-house vendoring

Reply:
It's not fragile. Embedding the SVG files directly in the binary should be quite robust.